CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 3.1.0 (Mar 29, 2022)
+
+FEATURES:
+
+* **New Resources:** Added following local repository resources in new implementation. [GH-378]
+  * "artifactory_local_cargo_repository"
+  * "artifactory_local_conda_repository"
```

Comment:
I don't see conda implemented?

Reply:
conda is similar to generic ... so added to repoTypesLikeGeneric list. No new code/file for conda.